packages/api/src/apps/datastore_api/app.ts

```diff
@@ -154,27 +156,6 @@ export default async (env: ProcessEnv) => {
     globalStats: Services.stats.Global({ debug }, appState),
   };
 
-  const initBlock = await provider.getBlock("latest");
-
-  // warm caches
-  await services.registry(appState.lastBlockUpdate, initBlock.number);
-  console.log("Got all EMP addresses");
-
-  await services.lspCreator.update(appState.lastBlockUpdate, initBlock.number);
-  console.log("Got all LSP addresses");
-
-  await services.emps.update(appState.lastBlockUpdate, initBlock.number);
-  console.log("Updated EMP state");
-
-  await services.lsps.update(appState.lastBlockUpdate, initBlock.number);
-  console.log("Updated LSP state");
-
-  // we've update our state based on latest block we queried
-  appState.lastBlockUpdate = initBlock.number;
-
-  await services.erc20s.update();
-  console.log("Updated tokens");
-
   // backfill price histories, disable if not specified in env
   if (env.backfillDays) {
```

Comment:
we might need to either get rid of this, or set some kind of flag where this doesnt run if there is already data in there. or maybe we just remmeber to remove the env once it runs once